Cargo.lock

```diff
@@ -669,8 +669,8 @@ dependencies = [
  "serde",
  "serde_json",
  "smallvec 1.11.0",
- "sp-arithmetic 16.0.0",
- "sp-core 21.0.0",
+ "sp-arithmetic 6.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This is really weird change, could you pls regenerate `Cargo.lock` and double check it?

Reply:
yes, thanks, didn't notice it, I didn't want to change anything in Cargo.lock at all!